charts/dependabot-gitlab/values.yaml

```diff
@@ -71,7 +71,7 @@ web:
     # -- Startup probe period
     periodSeconds: 10
     # -- Startup probe timeout
-    timeoutSeconds: 3
+    timeoutSeconds: 10
```

Comment:
There is no point in such a high timeout value during startup as service should not have when just booting. If service doesn't boot up in time, failureThreshold should be increased instead to keep pinging.

Reply:
if the service failes under load it will end up in a restart loop, because it can not start under load with such low probe timeouts

Reply:
The high values have opposite sideffect as well. If service is actually down, it will take significantly more time for it to be restarted.